src/util/file/file_test.go

```diff
@@ -92,6 +92,15 @@ func TestUserHome(t *testing.T) {
 	require.NotEqual(t, home, "")
 }
 
+func TestBuildDataDirDefault(t *testing.T) {
+	home := UserHome()
+	defaultDir := filepath.Join(home, ".skycoin")
+	dir, err := buildDataDir(defaultDir)
+	require.Nil(t, err)
+	expectedPath := filepath.Join(home, ".skycoin")
+	require.Equal(t, dir, expectedPath)
+}
```

Comment:
Are there tests for relative and absolute directory inputs too?

Reply:
This one could work for absolute path . I'll double check that we have test cases for relative paths .

Reply:
I would add an explicit absolute path test to be clear:

* One test using prefixed `UserHome()/.skycoin`
* One test using some path like `/opt/.skycoin`
* One test using some path like `../config/.skycoin`

Reply:
Some test cases have to be modified

Reply:
- Test for default path `UserHome()/.skycoin` implemented before in this issue
- Test for `/opt/.skycoin` added in 09f5832 
- Test for dot paths (e.g. `../config/.skycoin` ) was already there